hubspot3/engagements.py

```diff
@@ -120,3 +120,16 @@ def get_recently_modified(self, start_date: int, end_date: int, **options):
             engagements = clean_result("engagements", engagements, start_date, end_date)
 
             yield from engagements
+
+    def check_request(self, start_date: int):
```

Comment:
O nome `check_request` tem mais cara de verificar se ela é possível. Como essa função devolve quantos engagements foram alterados desde (since) uma data, ela deveria indicar isso no nome, por exemplo:
```suggestion
    def count_recently_modified_engagements(self, since: int):
```